WebApp/autoreduce_webapp/reduction_viewer/models.py

```diff
@@ -111,7 +111,7 @@ class ReductionRun(models.Model):
                                    on_delete=models.CASCADE)
     retry_run = models.ForeignKey('self', on_delete=models.SET_NULL, null=True, blank=True)
     status = models.ForeignKey(Status, blank=False, related_name='+', on_delete=models.CASCADE)
-    software = models.ForeignKey(Software, blank=False, related_name='reduction_runs', null=True,
+    software = models.ForeignKey(Software, blank=True, related_name='reduction_runs', null=True,
```

Comment:
A minor. I would suggest that adding a comment above this line saying something like "Allowed software field to be black for now and issued opened (#852) to later this field gets populated properly later".